styles.css

```diff
@@ -0,0 +1,296 @@
+* {
+    box-sizing: border-box;
+	margin:0; 
+}
+html {
+	font-size: 15px;
+}
+body {
+
+}
+nav {
+	width: 100%;
+	position: fixed;
+	top: 20px;
+}
+.centerNavBar {
+	background-color: #1c1c1c;
+	border-radius: 30px;
+	width: 1000px;
+	margin: 0 auto;
+	overflow: auto;
+	padding: 10px;
+}
+nav li{
+	list-style:none;
+}		
+a {
+	font-family: 'Lobster', 'cursive';
+	 float:left;
+	 color: #ffffff;
+	 text-decoration: none;
+	 margin-right: 90px;
+	 font-size: 1.5rem;
+} 
+nav ul {
+ 	 margin:0 auto;
+	 width:675px;
+}
+a:hover {
+	color:#b69a46;
+} 
+header {
+	width: 100%;
+	height:600px;
+ 	background-size:cover;
+ 	background-image: url("http://getwallpapers.com/wallpaper/full/b/8/8/654243.jpg");
+}
+header h1{
+	font-family: 'Allan'," cursive";
+	padding-top:270px;
+	text-align: center;
+	font-size:4rem;
+	color: #ffffff;		
+}
+.myName{
+	color:#b69a46;
+}
+section {
+	overflow:auto;
+	width:80%;
+	margin: 0 auto;
+}
+.titleAbout{
+	font-family: 'Lobster', 'cursive';
+	font-size:4rem;
+	padding:80px 120px;
+}
+.myImage{
+	float:left;
+    width:250px;
+	height:400px;
+	background-image: url("https://i.ibb.co/pJ9S9yf/eu.jpg");
+	background-repeat: no-repeat;
+	background-size:cover;
+	background-position:center center;
+}
+.aboutPar{
+	width:60%;
+	margin:40px 0px 200px 0px;
+	font-family: 'Allan'," cursive";
+	float:right;
+	font-size:2rem;	
+}
+article{
+background-color:#b69a46;
+width: 100%;	
+}	
+article h1{
+	font-size:4rem;
+	text-align:center;
+	padding-top:50px;
+	font-family: 'Lobster', 'cursive';
+}
+.upperContainer{
+	width:1000px;
+	height:300px;
+	margin:0 auto;
+	overflow:auto;
+}
+.lowerContainer{
+	width:1000px;
+	height:300px;
+	margin:0 auto;
+	overflow:auto;
+}
+.portBox{
+	float:left;
+	width:200px;
+	height:200px;
+	background:#ffffff;
+	margin-left: 40px;
+	margin-top: 50px;
+	border-radius:20px;
+	box-shadow: 10px 9px 12px 0px rgba(0,0,0,0.75);
+ 	background-size:cover;
+ 	background-image: url("http://how2girl.com/wp-content/uploads/2016/11/More-Coming-Soon.png");
```

Comment:
save a local copy of the image so you aren't relying on the image always being on someone elses website